### ISSUE TITLE
[Bug] - bcc package dependencies are incomplete

### ISSUE THREAD

**Issue by usui-tk** [amazonlinux/amazon-linux-2022#148]
**Describe the bug**

Conflicts and missing packages in the "amazonlinux" package repository in Amazon Linux 2022 prevent installation of "bcc" packages.

**To Reproduce**

```
[root@ip-172-31-13-162 ~]# cat /etc/image-id
image_name="al2022-ami"
image_version="2022"
image_arch="x86_64"
image_file="al2022-ami-2022.0.20220728.1-kernel-5.15-x86_64.xfs.gpt"
image_stamp="207c-c0b7"
image_date="20220728172608"
recipe_name="al2022 ami"
recipe_id="2caab3fc-cb8b-5512-6bfc-0939-7895-7e12-7d17ae5c"
```

```
[root@ip-172-31-13-162 ~]# python -c 'import dnf, json; db = dnf.dnf.Base(); print(json.dumps(db.conf.substitutions, indent=2))'
{
  "arch": "x86_64",
  "basearch": "x86_64",
  "releasever": "2022.0.20220728"
}
```

```
[root@ip-172-31-13-162 ~]# dnf install bcc --refresh
Amazon Linux 2022 repository
Error:
 Problem: conflicting requests
  - nothing provides libLLVM-12.so()(64bit) needed by bcc-0.18.0-3.amzn2022.x86_64
  - nothing provides libLLVM-12.so(LLVM_12)(64bit) needed by bcc-0.18.0-3.amzn2022.x86_64
  - nothing provides libclang-cpp.so.12()(64bit) needed by bcc-0.18.0-3.amzn2022.x86_64
(try to add '--skip-broken' to skip uninstallable packages)
```

**Expected behavior**

The package installs (works on 2022.0.20220728.1)

**Additional context**

I request verification of package dependencies, including the "bcc" package and the "bcc-tools" used.

```
[root@ip-172-31-13-162 ~]# dnf install bcc bcc-tools --refresh
Amazon Linux 2022 repository
Error:
 Problem 1: conflicting requests
  - nothing provides libLLVM-12.so()(64bit) needed by bcc-0.18.0-3.amzn2022.x86_64
  - nothing provides libLLVM-12.so(LLVM_12)(64bit) needed by bcc-0.18.0-3.amzn2022.x86_64
  - nothing provides libclang-cpp.so.12()(64bit) needed by bcc-0.18.0-3.amzn2022.x86_64
 Problem 2: package bcc-tools-0.18.0-3.amzn2022.x86_64 requires python3-bcc = 0.18.0-3.amzn2022, but none of the providers can be installed
  - package python3-bcc-0.18.0-3.amzn2022.noarch requires bcc = 0.18.0-3.amzn2022, but none of the providers can be installed
  - conflicting requests
  - nothing provides libLLVM-12.so()(64bit) needed by bcc-0.18.0-3.amzn2022.x86_64
  - nothing provides libLLVM-12.so(LLVM_12)(64bit) needed by bcc-0.18.0-3.amzn2022.x86_64
  - nothing provides libclang-cpp.so.12()(64bit) needed by bcc-0.18.0-3.amzn2022.x86_64
(try to add '--skip-broken' to skip uninstallable packages)
```

**Comment by stewartsmith** [comment]
We should be shipping an update to fix this shortly.

**Comment by usui-tk** [comment]
I confirm that the fix is in "Amazon-Linux-2022.0.2022.1207", so I am closing this Issue.
